credentials-template.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
This change is unnecessary.

Reply:
It might look silly, but materials which are supposed to cater to a wide range of audience should definitely pay attention to such details. I've been in trainings of size 50+ and wasting lot of time due to issues arising out of participants not giving enough attention to the order of keys and assigning them accordingly while they copy form developer accounts. So I think its a good idea to keep the order of keys in sync with the ones in the developer account so that there is less scope for a mismatch.